overrides/build-systems.json

```diff
@@ -1191,6 +1191,9 @@
     "setuptools",
     "setuptools-scm"
   ],
+  "argparse": [
+    "setuptools"
```

Comment:
[The last release of argparse was in 2015](https://pypi.org/project/argparse/) - are you sure you don't want to use the [argparse which is part of the Python standard library](https://docs.python.org/3/library/argparse.html) since Python 3.2?

Reply:
It unfortunately is a dependency of enowarschecker-[test, cli]. I will have a look what can be done upstream. After that I will update this PR.

Reply:
FYI: PR upstream is submitted: https://github.com/enowars/enochecker_cli/pull/19

Reply:
Upstream PR was merged. I removed the argparse entry. :)